### PR TITLE
set up database

### PR DESCRIPTION
**Pull request by reny1cao**

### CODE REVIEW

File: server/.env [hunk 1]
```diff
@@ -1 +1,3 @@
 TEAM_MEMBERS="Aecio, Bonnie, Renyi"
+
+MONGOLAB_URI="mongodb+srv://username:<password>@cluster0-ttytd.mongodb.net/test?retryWrites=true&w=majority"
```
Comment (tonyc856) [inline server/.env]: Looks good. At some point, I think we should remove the .env file from version control in the future

**Review by tonyc856 — APPROVED** [review]

File: server/.env (hunk 1 above)
Comment (reny1cao) [reply server/.env]: Yes. I was a bit confused about how to not push credentials on the github. The way I saw was have a local dev and save the password there, so I just put the general URI here. I believe you can create a user through mongodb and use it to connect to the database right?

Comment (tonyc856) [reply server/.env]: Yeah. And usually, we just have a local .env for local mongodb testing I believe. I'm confused on how we're going be using the mongodb we just made on atlas. I assume we're going to be using it as a database for production?

Comment (reny1cao) [reply server/.env]: I think we can also use it for building the project. Although I'm not sure what does the mongodb looks like on your side. I think you can create a new user and put the credentials in the URI and also whitelist your ip address, then you can connect to the cloud. It should works the same as a local database I think.

Comment (tonyc856) [reply server/.env]: Ah, I see. Got it.

**Pull request merged**